Formula/k/keploy.rb

```diff
@@ -3,20 +3,13 @@
 class Keploy < Formula
   desc "Testing Toolkit creates test-cases and data mocks from API calls, DB queries"
   homepage "https://keploy.io"
-  url "https://github.com/keploy/keploy/archive/refs/tags/v0.9.1.tar.gz"
-  sha256 "0e45207362bc17dd37e2efa8c881c72878525bca79ddfc45e6c242ea97fb12a1"
+  url "https://github.com/keploy/keploy/archive/refs/tags/v2.2.0-alpha23.tar.gz"
+  sha256 "eb21660c2f6a392d2c2f70115371a24a6ef767f1b04b813f6af02d8248c9622c"
   license "Apache-2.0"
 
   bottle do
     rebuild 1
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "be1248bf4f0a1b0824b8385bf78de390a761bb1c4c576687fd1a83eeef4d2017"
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "61adb55108c119cbaf60a0679b45a2bff5cce9d02e68e90b5f423fc7d0fe8f43"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "b3307c3128ee1a3d12d3c29d969af5dd27268aef707f3cc99cc69934c6b90156"
-    sha256 cellar: :any_skip_relocation, sonoma:         "06b885259fd6fc314d58b133749dbe7e2a9a7d660a304b1a298117d69d35ebc6"
-    sha256 cellar: :any_skip_relocation, ventura:        "57cf7126114751fd4b4e2bdf2a12d3820181fe069220a046fd1626a8b934fc02"
-    sha256 cellar: :any_skip_relocation, monterey:       "d8e8a354dfac6feedea50008b3b023e061dc4591c43ab7e1dd300913ff409c1d"
-    sha256 cellar: :any_skip_relocation, big_sur:        "7736b3ff430fa17c7d38311558adb2e3f790c771cb7ad841b56793cc0798eac4"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "e0709e89362133925d029532ce2fc6f0854d1e0c6cb2063b30fdfb4789b22ef7"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "8e4036d7f6e0eee31cd3fd0388f9e3e7b9167ecd8e278e156cf93c49c11a8601"
   end
```

Comment:
please dont touch bottle block, it would be handled by bot

Reply:
thanks for the help, have restored the bottle block and just update inital url and sha256 still some issues, simply want to update the formula to latest version